test/framework/easyconfig.py

```diff
@@ -2497,6 +2497,106 @@ def test_get_ref(self):
         self.assertEqual(descr_ref, "Toy C program, 100% %(name)s.")
         self.assertTrue(descr_ref is ec._config['description'][0])
 
+    def test_multi_deps(self):
```

Comment:
Test for multiple keys in `multi_deps`? Eg:

```
multi_deps = {
    'Python': ['2.7.15', '3.6.6'],
    'something_else': ['3.2']
}
```

I am not sure it is necessary, but you wrote the code for it (https://github.com/easybuilders/easybuild-framework/pull/2813/files#diff-0116faad5f652d385fd5806fe2ba0d51R1050), so we might as well test it

Reply:
enhanced the test in 3c34722